src/index.css

```diff
@@ -35,6 +45,15 @@ h1 {
   display: flex;
   justify-content: space-between;
   margin-bottom: 2rem;
+  background-color: #d6cff0;
+  padding: 20px 10px;
+}
+
+.recentScore h2 {
+  color: black;
+  animation-name: example;
+  animation-duration: 0.7s;
+  animation-iteration-count: infinite;
```

Comment:
- [ ] Nice work you did here 👏 . However, Kindly Move the `Leaderboard` heading to the left as it's aligned on the wireframe.  This gives a better UX and user view experience in the aspect of how well the contents are placed 👍 

## Wireframe:
![image](https://user-images.githubusercontent.com/50990925/225625665-dc738b57-e362-466a-a41e-192a0ad89ac9.png)

## Your view:
![image](https://user-images.githubusercontent.com/50990925/225625864-ff0e6069-7a20-4a34-9ccb-c5c6eae45f06.png)

Reply:
Suggested change has been implemented. Screen shot attached below for your reference.
![image](https://user-images.githubusercontent.com/117091923/225634811-19a011fd-c322-4139-91da-70df7254928e.png)

Reply:
Broken link to the License file has also been fixed. Screen shot attached below for your reference.
![image](https://user-images.githubusercontent.com/117091923/225638185-98783b2e-f5f7-4ec9-a965-68647673bf6c.png)
![image](https://user-images.githubusercontent.com/117091923/225638282-967571ac-26bf-4c09-9c89-dbaf26be84b1.png)